independent-publisher-connectors/OpenNEM/apiDefinition.swagger.json

```diff
@@ -0,0 +1,2456 @@
+                        "properties": {
+                          "id": {
+                            "type": "integer",
+                            "format": "int32",
+                            "description": "id"
+                          },
+                          "network": {
+                            "type": "object",
+                            "properties": {
+                              "code": {
+                                "type": "string",
+                                "description": "code"
+                              },
+                              "country": {
+                                "type": "string",
+                                "description": "country"
+                              },
+                              "label": {
+                                "type": "string",
+                                "description": "label"
+                              },
+                              "regions": {
+                                "type": "array",
+                                "items": {
+                                  "type": "object",
+                                  "properties": {
+                                    "code": {
+                                      "type": "string",
+                                      "description": "code"
+                                    }
+                                  }
+                                },
+                                "description": "regions"
+                              },
+                              "timezone": {
+                                "type": "string",
+                                "description": "timezone"
+                              },
+                              "timezone_database": {
+                                "type": "string",
+                                "description": "timezone_database"
+                              },
+                              "offset": {
+                                "type": "integer",
+                                "format": "int32",
+                                "description": "offset"
+                              },
+                              "interval_size": {
+                                "type": "integer",
+                                "format": "int32",
+                                "description": "interval_size"
+                              },
+                              "interval_shift": {
+                                "type": "integer",
+                                "format": "int32",
+                                "description": "interval_shift"
+                              },
+                              "intervals_per_hour": {
+                                "type": "number",
+                                "format": "float",
+                                "description": "intervals_per_hour",
+                                "title": ""

```

Comment:
@paulculmsee Empty property values can be removed until unless you have a specific reason to keep them.